frontend/src/CreateEvent/CreateEvent.module.css

```diff
@@ -0,0 +1,99 @@
+.container {
+  margin: 80px;
+  margin-right: 15vw;
```

Comment:
no margin-right is all good for now. other elements that parent this one will take care of that in the future